### PR TITLE
Bump Ruby to 2.7.8 & remember scan setting

### PR DESCRIPTION
**Pull request by TomNaessens**
This PR:
- bumps Ruby to 2.7.8
- updates some "background" gems to more recent versions/alternatives
- adds a development docker file
- switches the local development database to mariadb instead of sqlite
- updates the readme to include the `bin/dev` command instructions
- adds github workflows
- fixes the styling for the scan dropdowns
- remembers the last pick up/scan in setting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,10 @@ At an event, they can pick up the items at a pit by scanning their own barcode w
 
 ### Local installation
 * Clone this repository
-* Make sure Ruby 2.6.3 and Bundler 2.0 or higher are installed
+* Make sure the Ruby version defined in `.ruby-version`
```
Comment (NuttyShrimp) [inline README.md:27]: This sentence misses "is installed" at the end

File: README.md [hunk 2]
```diff
@@ -24,13 +24,27 @@ At an event, they can pick up the items at a pit by scanning their own barcode w
...
+* Make sure the Ruby version defined in `.ruby-version` is installed (using `asfd`, just run `asdf install`) 
 * Run `bundle install`
-* Run `rails s`
+* Run `bin/dev` to start a database docker image
+* Run `bundle exec rake db:seed` to set up some basic data
+* Run `bundle exec rails s` to start the Rails server
 * Visit `http://localhost:3000`
+* Sign in using `saruman@zeus.ugent.be` and `thewhite` as password
 
 ### Deploy update to server
```
Comment (TomNaessens) [inline README.md:35]: Anyone that can verify this is correct? Didn't dare to test it out yet 😅

**Review by FKD13 — APPROVED** [review]
Snel nagekeken, Ik zie geen dingen die me direct twijfelachtig lijken :D

File: README.md (hunk 2 above)
Comment (TomNaessens) [reply README.md:35]: Dit gaan we wel merken

**Pull request merged**